content/en/blog/2024/collecting-otel-compliant-java-logs-from-files/index.md

```diff
@@ -0,0 +1,379 @@
+---
+title: Collecting OpenTelemetry-compliant Java logs from files
+linkTitle: OTel-compliant Java logs from files
+date: 2024-12-03
```

Comment:
```suggestion
date: 2024-12-07
```

Since we have a queue of blog posts right now, we will ship fifo, so https://github.com/open-telemetry/opentelemetry.io/pull/5575 comes first, this one second. I recommend that we put ~2days between publishing the two